lib/engine.py

```diff
@@ -654,15 +664,21 @@ def insert_statement(self, values):
         columncount = len(self.table.get_insert_columns(join=False, create=False))
         insert_stmt = "INSERT INTO " + self.table_name()
         insert_stmt += " (" + columns + ")"
-        insert_stmt += "VALUES ("
-        for i in range(0, columncount):
-            insert_stmt += "%s, "
-        insert_stmt = insert_stmt.rstrip(", ") + ");"
-        n = 0
-        while len(values) < insert_stmt.count("%s"):
-            values.append(self.format_insert_value(None, types[n]))
-            n += 1
-        insert_stmt %= tuple([str(value) for value in values])
+        insert_stmt += " VALUES " 
+        vals = ""
+        for rows in values:
+            valk= rows
+            insert_stmt2 = " ("
+            for i in range(0, columncount):
+                insert_stmt2 += "%s, "
+            insert_stmt2 = insert_stmt2.rstrip(", ") + "),"
+            n = 0
+            while len(valk) < insert_stmt.count("%s"):
+                valk.append(self.format_insert_value(None, types[n]))
+                n += 1
+            insert_stmt2 %= tuple([str(value) for value in valk])
+            insert_stmt += insert_stmt2
```

Comment:
So, if I'm reading things right, `insert_stmt` at this point is:

`"INSERT INTO " + self.table_name() + " (" + columns + ")" + " VALUES "  + insert_stmt2`

What do you think about calling `insert_values`, deleting all of the `insert_stmt` code before this point, and the changing this line to:

`insert_stmt = "INSERT INTO {} ({}) VALUES {}".format(self.table_name(), columns, insert_values)`

My goal with this is to decrease the complexity and increase the readability.